pkg/apis/kops/componentconfig.go

```diff
@@ -235,9 +235,9 @@ type KubeletConfigSpec struct {
 	// The CIDR to use for pod IP addresses, only used in standalone mode.
 	// In cluster mode, this is obtained from the master.
 	PodCIDR string `json:"podCIDR,omitempty" flag:"pod-cidr"`
-	//// ResolverConfig is the resolver configuration file used as the basis
-	//// for the container DNS resolution configuration."), []
-	//ResolverConfig string `json:"resolvConf"`
+	// ResolverConfig is the resolver configuration file used as the basis
+	// for the container DNS resolution configuration."), []
+	ResolverConfig *string `json:"resolvConf" flag:"resolv-conf"`
```

Comment:
I've tested this on 1.6.1. I don't think we need to add a check for this. It seems that the kubelet has supported this flag for a while. The earliest mention of resolv-conf that I could find was [here](https://github.com/kubernetes/kubernetes/commit/1162cfd3b10b3b18aaa8b3b476beea9b1738bdb4) which was from version `v1.2.0-alpha.7`.